lab_1_classify_by_unigrams/main.py

```diff
@@ -6,11 +6,22 @@
 
 def tokenize(text: str) -> list[str] | None:
     """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+       Splits a text into tokens, converts the tokens into lowercase,
+       removes punctuation, digits and other symbols
+       :param text: a text
+       :return: a list of lower-cased tokens without punctuation
+       """
+    if type(text) != str:
+        return None
+    text = text.lower()
+    tokens = []
+    for token in text:
+        if token.isalpha() and token != 'º':
```

Comment:
where did you find this symbol? And for what?

Reply:
> where did you find this symbol? And for what?

It was in the English text, it's the degree sign. It is considered a letter, so my code kept counting it.

Reply:
fixed